plugins/bulk-import/src/components/Repositories/RepositoriesList.tsx

```diff
@@ -1,48 +1,49 @@
 import React from 'react';
 import { useLocation, useNavigate } from 'react-router-dom';
 
-import { ErrorPage, Table } from '@backstage/core-components';
+import { ErrorPage } from '@backstage/core-components';
```

Comment:
Maybe just as a learning for me, why couldn't we use Backstage core-components Table here anymore?